web/src/components/core/IssuesDrawer.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { forwardRef } from "react";
+import {
+  HelperText,
+  HelperTextItem,
+  NotificationDrawer,
+  NotificationDrawerBody,
+  NotificationDrawerHeader,
+  Stack,
+} from "@patternfly/react-core";
+import Link from "~/components/core/Link";
+import { useAllIssues } from "~/queries/issues";
+import { useInstallerStatus } from "~/queries/status";
+import { IssueSeverity } from "~/types/issues";
+import { InstallationPhase } from "~/types/status";
+import { _ } from "~/i18n";
+
+/**
+ * Drawer for displaying installation issues
+ */
+const IssuesDrawer = forwardRef(({ onClose }: { onClose: () => void }, ref) => {
+  const issues = useAllIssues();
+  const { phase } = useInstallerStatus({ suspense: true });
+  const { issues: issuesByScope } = issues;
+
+  // FIXME: share below headers with navigation menu
+  const scopeHeaders = {
+    users: _("Users"),
+    storage: _("Storage"),
+    software: _("Software"),
+  };
+
+  if (issues.isEmpty || phase === InstallationPhase.Install) return;
+
+  return (
+    <NotificationDrawer ref={ref}>
+      <NotificationDrawerHeader title={_("Preflight checks")} onClose={onClose} />
```

Comment:
Reviewer, please note the use of "Preflight checks" to avoid the "Issues" technicality. A first attempt was to use "Installation issues", but these are not actually "Installation issues" since the installation does not even started. It is more an "Installation settings issues", but sounds difficult to explain. Looks like "Preflight checks" could be a more self-explanatory _term_. As usual, better ideas are more than welcome.

Reply:
"Preflight checks" means something to me since I used flight sims a lot 1995-2005, but I am not sure if a normal user can make sense of this; much less one with limited English knowledge.

I don't even dare to think about translations and how much they make sense to users.

Please let's not get overly creative with this; keep it simple.

Reply:
@shundhammer 

What do you propose?

Reply:
How about "consistency checks"?

Try to avoid lingo from a completely different area of technology like aviation.

Reply:
Not sure about the word "consistency". Sounds weird to me in this context. We could go for "Installation checks" or "Settings checks", but both are too generic IMHO

Reply:
Let's use "Pre-installation checks"